improver/metadata/amend.py

```diff
@@ -66,6 +68,22 @@ def update_stage_v110_metadata(cube: Cube) -> None:
     cube.attributes["mosg__grid_version"] = "1.1.0"
 
 
+def _load_json(json_path: str) -> Dict:
+    """Load a JSON file into a dictionary.
+
+    Args:
+        json_path:
+            Path to the JSON file to load.
+
+    Returns:
+        Dictionary containing the JSON data.
+    """
+    if not isinstance(json_path, str):
```

Comment:
Does this preclude the json_path being a pathlib-Path object? Is that desirable going forwards?

Reply:
Good point, I'll take a look.